asar-pi-applications/asar_search_algorithm/A_Star_Search.py

```diff
@@ -208,7 +207,7 @@ def path_to_move(path, tile):
 
 
 def main():
-    go_start, go_goal, new_tile = cfg.give_dng(inst_graph())
+    go_start, go_goal, new_tile = cfg.give_dng(instantiate_graph())
```

Comment:
What is `dng`? Is it danger? Needs to be more clear

Reply:
resolved